### DISCUSSION TITLE
Checkup fails on [✗] Build hello-skip (22.22s)

### DISCUSSION THREAD

**Discussion by markonikolov16** [skiptools/skip#156] [Q&A] [answered]
Seems like Im the only one with the following error: 

 <unknown>:0: warning: compiler plugin not loaded: '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/bin/swift-plugin-server; failed to initialize
/private/var/folders/d4/zddmcc897852mjx0lmmxn1km0000gn/T/EFD00212-1DF6-44CF-8CDB-C0251C431A03/hello-skip/Sources/HelloSkip/ContentView.swift:80:1: error: external macro implementation type 'PreviewsMacros.SwiftUIView' could not be found for macro 'Preview(_:body:)'
#Preview {
^
SwiftUI.Preview:2:41: note: 'Preview(_:body:)' declared here
@freestanding(declaration) public macro Preview(_ name: String? = nil, body: @escaping @MainActor () -> any View) = #externalMacro(module: "PreviewsMacros", type: "SwiftUIView")
                                        ^
rosetta error: Failed to open libRosettaRuntime
 <unknown>:0: warning: compiler plugin not loaded: '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/bin/swift-plugin-server; failed to initialize
/private/var/folders/d4/zddmcc897852mjx0lmmxn1km0000gn/T/EFD00212-1DF6-44CF-8CDB-C0251C431A03/hello-skip/Sources/HelloSkip/ContentView.swift:80:1: error: external macro implementation type 'PreviewsMacros.SwiftUIView' could not be found for macro 'Preview(_:body:)'
#Preview {
^
SwiftUI.Preview:2:41: note: 'Preview(_:body:)' declared here
@freestanding(declaration) public macro Preview(_ name: String? = nil, body: @escaping @MainActor () -> any View) = #externalMacro(module: "PreviewsMacros", type: "SwiftUIView")
                                        ^
error: fatalError

**Comment by markonikolov16** [comment, marked as answer]
To all my homies having this issue, make sure you disable 'Build using Rosetta' for your terminal. This now has led me to a different error regarding the Java version.

**Reply by marcprux** [reply]
Can you let us know what the new error is?

**Reply by markonikolov16** [reply]
it was solved in this thread:
https://github.com/orgs/skiptools/discussions/157#discussioncomment-9887091
